01-rstudio-intro.Rmd

```diff
@@ -438,18 +438,31 @@ network). R and RStudio have functionality for managing packages:
 >
 
 > ## Challenge 2 {.challenge}
+> Which of the following are valid R variable names?
+> ```{r, eval=FALSE}
+> min_height
+> max.height
+> _age
+> .mass
+> MaxLength
+> min-length
+> 2widths
+> celsius2kelvin
+> ```
+
+> ## Challenge 3 {.challenge}
 >
 > Run the code from the previous challenge, and write a command to
```

Comment:
This needs to be re-phrased "Run the code from challenge 1 ...", alternately, my preference would be to swap new challenge 2 with old challenge 1.